sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -770,10 +770,11 @@
           <configuration>
             <skip>${spotbugs.skip}</skip>
             <effort>max</effort>
-            <threshold>Low</threshold>
+            <threshold>low</threshold>
```

Comment:
Is this case sensitive?

Reply:
No, but I wanted to see if it was. I'm just going to leave it given how long Core - CI takes